test/dartdoc_test.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library dartdoc.dartdoc_test;
+
+import 'dart:io';
+
+import 'package:unittest/unittest.dart';
+
+import '../lib/dartdoc.dart';
+import '../lib/src/package_utils.dart';
+
+
+tests() {
+  group('dartdoc test', () {
+
+    test('version info', () {
+      String version = getPackageVersion(Directory.current.parent.path);
```

Comment:
The tests generally run from the project root - I think you want this to be `Directory.current.path`.